src/launch/launch.html

```diff
@@ -6,10 +6,8 @@
         <div>Apply to host your Launch</div>
       </div>
       <div class="subtext">
-        <div>
-          <div>PrimeLAUNCH empowers and creators to host and fair and</div>
-          <div>seemless launch in a completely permissionless setting.</div>
-        </div>
+        PrimeLAUNCH empowers builders and creators to host a fair and
+        <br /> seemless launch in a completely permissionless setting.
```

Comment:
Please remove this BR, nor split with divs.  This is not a good practice except under special circumstances.  Just let the text flow as it will.  Shrink the container if you must.

Reply:
Okay